### PR TITLE
Expose dev instances

### PR DESCRIPTION
**Pull request by maximusunc**
Removed redundant ingress annotation code from services

### CODE REVIEW

File: helm/strider/values.yaml [hunk 1]
```diff
@@ -90,10 +90,13 @@ redis:
     - 'no'
 
 ingress:
+  enabled: true
```
Comment (PhillipsOwen) [inline helm/strider/values.yaml:93]: this definitely should not be here

File: helm/strider/values.yaml [hunk 2]
```diff
@@ -109,11 +112,15 @@ ingress:
...
     host: "the kp host name"
     annotations:
+      cert-manager.io/cluster-issuer: letsencrypt
+      nginx.ingress.kubernetes.io/whitelist-source-range: "0.0.0.0/0,::/0"
       nginx.ingress.kubernetes.io/proxy-body-size: "250m"
       nginx.ingress.kubernetes.io/proxy-connect-timeout: "86400"
       nginx.ingress.kubernetes.io/proxy-read-timeout: "86400"
       nginx.ingress.kubernetes.io/proxy-send-timeout: "86400"
   pathType: "ImplementationSpecific"
+  tls:
```
Comment (PhillipsOwen) [inline helm/strider/values.yaml:122]: nor should this be here

**Review by PhillipsOwen — CHANGES_REQUESTED** [review]
the values.yaml files should only have values that are non-specific to the development or production product deployments.

**Review by PhillipsOwen — CHANGES_REQUESTED** [review]
see notes I left in slack

**Review by PhillipsOwen — CHANGES_REQUESTED** [review]
in answer coalesce dev values file: if ingress and TLS is enabled, and a cert-manger and a white-list values are specified then the server host name should be https://.  insure ncats meta.yaml has correct image/tag values.

in aragorn ranker dev/prod values files:  please review image tag values and note the differences.

in robokop-ara values files. replace pod security context with {} in values.yaml, other values files should have the real values.

in strider values file. xtrapi values indicate production settings. just blank them out and the the dev/prod values continue to be the real values.

**Comment by maximusunc** [comment]
Made all requested changes. All the ncats image versions looked correct.

**Pull request merged**